provisioning/templates-api/start_minikube.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -Eeuo pipefail
+
+echo
+echo "Starting minikube ..."
+echo "--------------------------"
+
+# 1. Use v1.24 - https://github.com/kubernetes/minikube/issues/15173
+# 2. Enable Network Policies support.
+#    Kindnet - default CNI, does not support Network Policies, by design.
+#    https://minikube.sigs.k8s.io/docs/handbook/network_policy/
+minikube start \
+--wait "apiserver,system_pods,default_sa,apps_running,node_ready,kubelet" \
+--kubernetes-version=v1.24.7 \
+--cni=calico
```

Comment:
- `--wait`, niektore casti `MiniKube` startuju dhlsie - nase app pods sa zbytocne restartovali (nebolo potom jasne ci je to chyba alebo nie)
- `--kubernetes-version` - `v1.25` ma nejaky bug s `Calico` https://github.com/kubernetes/minikube/issues/15173
- `--cni=calico` MiniKube by default ignoruje `NetworkPolicy`, `calico` CNI ich podporuje, ... teda som to zapol, aby sa dalo co najviac veci otestovat lokalne a v CI